memcache.py

```diff
@@ -1523,4 +1528,4 @@ def _doctest():
         sys.exit(1)
 
 
-# vim: ts=4 sw=4 et :
+# vim: ts=4 sw=4 et :
```

Comment:
this is missing a new line which makes it an invalid file in POSIX, that's why it failed travis on python 2.6